.circleci/config.yml

```diff
@@ -258,123 +81,51 @@ jobs:
-      - attach_workspace:
-          at: .
-      - run: *install_source_dependencies
-      - run: *build
-      - run: *install
-      - run:
-          name: "Copy plugin to artifact folder"
-          command: |
-            mkdir -p /tmp/artifacts
-            cp /tmp/build/usr/local/lib/libdd_opentracing_plugin.so /tmp/artifacts/libdd_opentracing_plugin.so
-      - store_artifacts:
-          path: /tmp/artifacts
-
-  integration_test_nginx_ubuntu_18_04:
-    <<: *defaults
-    <<: *image_ubuntu_18_04
+      NGINX_OPENTRACING_VERSION: 0.4.0
+      NGINX_VERSION: 1.14.0
     steps:
+      - checkout
       - attach_workspace:
           at: .
-      - run: *install_nginx
       - run:
-          name: Install Datadog OpenTracing nginx module
+          name: Install nginx
           command: |
             CODENAME=$(lsb_release -c | cut -f2) # eg "bionic", "xenial"
-            apt install -y ./artifacts/datadog-opentracing-nginx*${CODENAME}*.deb
-            ldconfig
-            NGINX_CONF=$(nginx -V 2>&1 | grep "configure arguments" | sed -n 's/.*--conf-path=\([^ ]*\).*/\1/p')
-            # Different nginx configs have different webroots for the sample page. Make sure our
-            # config points to the right one.
-            WEB_ROOT=$(grep -rh '^\s*root' /etc/nginx/ | head -1)
-            cp test/integration/nginx/nginx.conf $NGINX_CONF
-            sed -i "s|$(grep -rh '^\s*root' ${NGINX_CONF} | head -1)|${WEB_ROOT}|g" ${NGINX_CONF}
+            wget http://nginx.org/keys/nginx_signing.key
+            apt-key add nginx_signing.key
+            echo deb http://nginx.org/packages/ubuntu/ ${CODENAME} nginx >> /etc/apt/sources.list
+            echo deb-src http://nginx.org/packages/ubuntu/ ${CODENAME} nginx >> /etc/apt/sources.list
+            apt-get update
+            apt-get install nginx=${NGINX_VERSION}-1~${CODENAME}
       - run:
-          name: Run tests
+          name: Integration test
           command: |
+            # Install the Datadog plugin
+            cp ./build/libdd_opentracing_plugin.so /usr/local/lib/libdd_opentracing_plugin.so
+            # Change the config to use it.
+            NGINX_MODULES=$(nginx -V 2>&1 | grep "configure arguments" | sed -n 's/.*--modules-path=\([^ ]*\).*/\1/p')
+            wget https://github.com/opentracing-contrib/nginx-opentracing/releases/download/v${NGINX_OPENTRACING_VERSION}/linux-amd64-nginx-${NGINX_VERSION}-ngx_http_module.so.tgz
+            tar zxf linux-amd64-nginx-${NGINX_VERSION}-ngx_http_module.so.tgz -C ${NGINX_MODULES}
             cd ./test/integration/nginx/
+            NGINX_CONF=$(nginx -V 2>&1 | grep "configure arguments" | sed -n 's/.*--conf-path=\([^ ]*\).*/\1/p')
+            cp nginx.conf $NGINX_CONF
+            cp dd-config.json /etc/dd-config.json
+            mkdir -p /var/www/
+            cp index.html /var/www/
+            # Run the tests.
             ./nginx_integration_test.sh
 
 
-_workflow_filters:
```

Comment:
If a new release-only job step is added in the future, all of this will need to be added again, right?
Maybe keep it unless the plan is to not need it in the future.

Reply:
ehhh, it can live on in the git history I reckon